scripts/install.js

```diff
@@ -131,37 +131,50 @@ function getProxy() {
  */
 
 function checkAndDownloadBinary() {
-  if (sass.hasBinary(sass.getBinaryPath())) {
+  if (process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI) {
+    console.log('Skipping downloading binaries on CI builds');
     return;
   }
+  var binaryPath = sass.getBinaryPath();
 
-  mkdir(path.dirname(sass.getBinaryPath()), function(err) {
+  if (sass.hasBinary(binaryPath)) {
+    return;
+  }
+
+  mkdir(path.dirname(binaryPath), function(err) {
     if (err) {
       console.error(err);
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
-
-      console.log('Binary downloaded and installed at', sass.getBinaryPath());
-    });
+    var cachePath = path.join(sass.getCachePath(), pkg.name, pkg.version);
+    var cacheBinary = path.join(cachePath, sass.getBinaryName());
+    if (!fs.existsSync(cacheBinary)) {
```

Comment:
nit: avoid negative conditional with an else branch. Inverting this `if` makes it much easier to read.